src/main/java/ybe/mini/travelserver/domain/room/service/RoomService.java

```diff
@@ -18,8 +18,8 @@ public class RoomService {
 
     @Transactional(readOnly = true)
     public List<RoomGetResponse> bringRooms(Long accommodationId) {
-        List<Room> rooms = roomRepository.findByAccommodationId(accommodationId);
-        return rooms.stream()
+        return roomRepository.findByAccommodationId(accommodationId)
+                .stream()
                 .map(RoomGetResponse::fromEntity)
                 .collect(Collectors.toList());
```

Comment:
java 17 에서는 `.collect(Collectors.toList())` 대신
`.toList()` 도 지원됩니당

collect() 에서 부가적인 프로세스가 발생하기 때문에 
직접 사용하는 것을 권장하고 있어요.
sonarLint 플러그인 설치하시면 도움이 많이 되실 겁니다 😊

> java:S6204
In Java 8 Streams were introduced to support chaining of operations over collections in a functional style. The most common way to save a result of such chains is to save them to some collection (usually List). To do so there is a terminal method collect that can be used with a library of Collectors. The key problem is that .collect(Collectors.toList()) actually returns a mutable kind of List while in the majority of cases unmodifiable lists are preferred. In Java 10 a new collector appeared to return an unmodifiable list: toUnmodifiableList(). This does the trick but results in verbose code. Since Java 16 there is now a better variant to produce an unmodifiable list directly from a stream: Stream.toList().
This rule raises an issue when "collect" is used to create a list from a stream.